packages/common/src/Logger.ts

```diff
@@ -1,11 +1,50 @@
 /**
- * Single method to be used for logging purpose.
+ * Single class to be used for logging purpose.
  *
  * @param {any} message Message to be logged
  */
-export function _logMessage(message: any) {
-  // Method would be called ifLogsEnabled is true
-  if (console.log) {
-    console.log(message)
+class Logger {
+  // By default, the logger is not in debug mode.
+  static isDebug: boolean =
+    process.env.BICONOMY_SDK_DEBUG === 'true'
+      ? true
+      : process.env.REACT_APP_BICONOMY_SDK_DEBUG === 'true'
+      ? true
+      : false
+
```

Comment:
Here checking two variables, either developer can se `BICONOMY_SDK_DEBUG` or `REACT_APP_BICONOMY_SDK_DEBUG` true.

This is because in backend we can set the env variable and use the env variable, but for frontend developer have two choises
-  Use dotenv to resolve the local .env variable
- Use the REACT_APP_ prefix variable as in `create-react-app` by default [reads all the env variables which starts with REACT_APP_](https://stackoverflow.com/a/48378498/11097431)